vendor/bat-native-ads/src/bat/ads/internal/client_mock.h

```diff
@@ -33,6 +33,11 @@ class ClientMock : public Client {
       const std::string& campaign_id,
       const int64_t time_offset_per_ad_in_seconds,
       const uint8_t count);
+      
+  void GeneratePastPurchaseIntentSignalHistoryFromNow(
+    const std::string segment,
+    const uint64_t time_offset_in_seconds,
+    const std::uint16_t weight);
```

Comment:
nit: Can we remove `std::`

Reply:
also changing `const std::string segment` to `const std::string& segment`